### PR TITLE
Fixing lint issues

### PR DESCRIPTION
**Pull request by MrJuneJune**
- Fixed issues across the files such as variable names and missing doc strings.

### CODE REVIEW

File: jurassic-journalists/main.py [hunk 1]
```diff
@@ -44,57 +49,58 @@ class Pili(Image):
     sin_wobble_speed = NumericProperty(0)
     """
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.img = Im.new("RGBA", (720,720), (200,200,200,0)) # Im.open('paper.png') #self.size
+        super(TextPaper, self).__init__(*args, **kwargs)
```
Comment (Numerlor) [inline jurassic-journalists/main.py]: Arguments not needed here

File: jurassic-journalists/classes.py [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
     def __init__(self,char,size,font,color = (255,255,255,255),b=False,i=False,u=False):
+        """
+        char: character.
+        size: size of letter.
+        font: PIL truetype font object. TODO: add handling for other types
+        color: color of letter, RGBA tuple, range 0-1.
+        b: Bold flag.
+        i: Italics flag.
+        u: Underlined flag.
+        """
```
Comment (Numerlor) [inline jurassic-journalists/classes.py:13]: For the future, are we going to use `typing` for params or do the docstrings like this?

Comment (MrJuneJune) [reply jurassic-journalists/classes.py:13]: You can use both typing and doc string. `typing` is not used to describe a method rather used so what `type` of variable is being accepted. In here, doc string is used to describe what each variable means. http://google.github.io/styleguide/pyguide.html
image

File: jurassic-journalists/main.py (hunk 1 above)
Comment (MrJuneJune) [reply jurassic-journalists/main.py]: Thanks !  Will update

File: jurassic-journalists/classes.py (hunk 2 above)
Comment (Numerlor) [reply jurassic-journalists/classes.py:13]: yep just that the whole signature doesn't need to be explained when we just throw in the typehints; like for char and size above

File: jurassic-journalists/classes.py [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
-        #gets dimensions as tuple(w,h) that it will be when rendered.
+
+    def get_kerning(self):
+        """ gets dimensions as tuple(w,h) that it will be when rendered. """
         return self.font.getsize(self.char)
         
-
-#Witeout class- the little squiggle you have to do to erase words.
 class WiteOut:
+    """ Witeout class- the little squiggle you have to do to erase words. """
```
Comment (Numerlor) [inline jurassic-journalists/classes.py:27]: Is this a typo? (WiteOut vs WhiteOut)

Comment (HypoT) [reply jurassic-journalists/classes.py:27]: Its a brand name - either will do.

**Comment by HypoT** [comment]
Merge away, I'll try & not push further linting errors

**Review by HypoT — APPROVED** [review]

**Pull request merged**